src/SmartComponents/modules/reducers.js

```diff
@@ -11,11 +11,12 @@ const initialState = {
     factFilter: '',
     totalFacts: 0,
     page: 1,
+    sort: 'none',
     perPage: 10,
     loading: false
 };
 
-function paginateFilteredData(data, selectedPage, factsPerPage) {
+function paginateSortedData(data, selectedPage, factsPerPage) {
```

Comment:
This function doesn't care if the data is sorted or filtered right? so this could just be `paginateData` ?